portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +24,18 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h3>Hello Andrew!</h3>");
+    String[] ArrayList = {"Hey!", "Check", "here!"};
```

Comment:
Nit: (1) please use camel case with a starting lower case for variable; (2) please use a more specific name for variables, e.g., wordArray, or something else if you prefer.

Also ArrayList is a core Java class. Even for a class name, you would want to avoid defining a class with this exact name.